src/app/features/home/user-navbar/navbar.component.html

```diff
@@ -21,7 +21,7 @@
   <div class="main-container" [ngClass]="menu ? 'nav-menu' : 'nav-menu--hide'">
     <ul class="nav-menu__list" *ngFor="let value of navbarList[0]; let i = index">
       <li>
-        <a class="nav-menu__link" routerLink="/{{ navbarList[1][i] }}">{{ value }} </a>
+        <a class="nav-menu__link" (click)="showMenu()" routerLink="/{{ navbarList[1][i] }}">{{ value }} </a>
```

Comment:
jeżeli element jest klikalny a nie jest tylko hiperłączem to powinnien być to button. Logikę przekierowania wtedy dodajesz wewnątrz handlera na click

Reply:
Zmieniłem `<a>` na button.